packages/editor/src/store/reducer.js

```diff
@@ -1203,13 +1203,29 @@ export function autosave( state = null, action ) {
 				title,
 				excerpt,
 				content,
-				preview_link: post.preview_link,
 			};
+	}
+
+	return state;
+}
+
+/**
+ * Reducer returning the poost preview link
+ *
+ * @param  {string?} state  The preview link
+ * @param  {Object} action Dispatched action.
+ *
+ * @return {string?} Updated state.
+ */
+export function previewLink( state = null, action ) {
```

Comment:
Preview link is tracked separately as it's not directly related to autodrafts anymore.